src/js/base/module/Clipboard.js

```diff
@@ -18,12 +18,19 @@ export default class Clipboard {
   pasteByEvent(event) {
     const clipboardData = event.originalEvent.clipboardData;
     if (clipboardData && clipboardData.items && clipboardData.items.length) {
-      // paste img file
       const item = clipboardData.items.length > 1 ? clipboardData.items[1] : lists.head(clipboardData.items);
       if (item.kind === 'file' && item.type.indexOf('image/') !== -1) {
+        // paste img file
         this.context.invoke('editor.insertImagesOrCallback', [item.getAsFile()]);
+        this.context.invoke('editor.afterCommand');
+      } else if (item.kind === 'string') {
+        // paste text check maxTextLength
+        let text = clipboardData.getData('Text');
```

Comment:
Could you merge L28 ~ L30 into one line?
e.g.: `if (this.context.invoke(...)) {`

Reply:
And `this.context.invoke('editor.afterCommand');` should be called after pasting text successfully.